crypto/cmp/cmp_int.h

```diff
@@ -0,0 +1,591 @@
+/*
+ * Copyright 2007-2018 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright Nokia 2007-2018
+ * Copyright Siemens AG 2015-2018
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ *
+ * CMP implementation by Martin Peylo, Miikka Viljanen, and David von Oheimb.
+ */
+
+#ifndef OSSL_HEADER_CMP_INT_H
+# define OSSL_HEADER_CMP_INT_H
+
+# define DECIMAL_SIZE(type) ((sizeof(type)*8+2)/3+1) /* this avoids dependency
+                      on internal header file: #include "internal/cryptlib.h" */
```

Comment:
Put the comment on the line before. Please use spaces between the digits and operators. Why do we care if there is a dependency on cryptlib.h?

Reply:
I've removed the dependency on cryptlib.h because `DECIMAL_SIZE` has been the only reason to include it and not referring the internal header makes life easier for people currently using our code in standalone fashion.

Reply:
So we'll replace this `# define `... by `# include "internal/cryptlib.h"`.

Reply:
Done.